src/layouts/shortcodes/subctl-install.html

```diff
@@ -0,0 +1,15 @@
+Download the subctl binary and make it available on your PATH.
+
+<!-- TODO: create a "getsubctl.sh" to simplify this -->
+
+<div class="highlight">
+<pre style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4">
+<code class="language-bash" data-lang="bash">
+mkdir -p ~/.local/bin
+curl -L -o ~/.local/bin/subctl https://github.com/submariner-io/submariner-operator/releases/download/v0.1.0/subctl-v0.1.0-linux-amd64
```

Comment:
As we are going to release 0.1.1 shortly, do you want to update this?

Reply:
I'll do on separate patch :-) thanks